dockerfiles/base/tests/Dockerfile

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
there is already dockerfiles/test folder and it has different Dockerfile. Having dockerfiles/tests and dockerfiles/test would be ambiguous.